app/assets/stylesheets/hyrax/_collection_types.scss

```diff
@@ -47,3 +47,7 @@
     }
   }
 }
+
+.collection_type_share_applies_to {
+  margin-left: 30px;
```

Comment:
I've heard @mtribone say we should use `em`s rather than `px`.  Is that right @mtribone?

Reply:
Yes relative units like `em` or percent are better than absolute units like pixels. It helps facilitate better scaling when increasing the font sizes or the elements on the screen for better usability. If everything is a relative unit, they scale together much more nicely.